### PR TITLE
:memo: FC-1158 New documentation for getting started

### PR DESCRIPTION
**Pull request by Templum**

### CODE REVIEW

**Review by chrbrt — CHANGES_REQUESTED** [review]

File: pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/configure.md [hunk 1]
```diff
@@ -11,12 +11,234 @@ indicator: both
...
+    // Accessing an environment variable and parsing it to ensure proper type
+    const myBackendVersion = Number.parseInt(process.env.MY_BACKEND_VERSION, 10);
+```
+
+### Secrets
+
+{: .notice}
+Please be aware that we discourage the storing of credentials like password. Instead we recommend credentials used for token-based authentication methods.
+
+We offer a secret store based on Vault that is available to functions at runtime. And allow access to a pool of account specific secrets. There are also some default secrets, that are created as part of the [provisioning](liveperson-functions-provisioning.html).
```
[the remark this answers is not in the document]
Comment (Templum) [reply pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/configure.md:69]: Actually, you have a legit point here, let me remove it. Idea was to give them a sense of security, given Vault is a battle-tested solution for secret storage.

File: pages/Documents/DeveloperTools/LivePersonFunctions/Foundations/liveperson-functions-cli.md [hunk 2]
```diff
@@ -12,13 +12,127 @@ redirect_from:
...
+--header 'Accept: application/json' \
+--data-raw '{
+   "username":  "<BOT LOGIN NAME>",
+   "appKey": "<API KEY>",
+   "secret": "<API SECRET>",
+   "accessToken": "<ACCESS TOKEN>",
+   "accessTokenSecret": "<ACCESS SECRET>"
+}'
+```
+To get the correct domain for your account use the [Domain Retrieval tool](https://developers.liveperson.com/agent-domain-domain-api.html) select the value for agent-vep from the list. Fill out all the other fields with data from your account.
```
Comment (Templum) [inline pages/Documents/DeveloperTools/LivePersonFunctions/Foundations/liveperson-functions-cli.md:98]: We should avoid using full domains when we reference to developer stuff (as we don't use it for our links)

Comment (chrbrt) [reply pages/Documents/DeveloperTools/LivePersonFunctions/Foundations/liveperson-functions-cli.md:98]: done

File: pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/deep-dive-cli.md [hunk 3]
```diff
@@ -10,15 +10,212 @@ indicator: both
...
+If you want to invalidate the login information a `lpf logout` will do the trick. There are situations where you want to completely delete the login information (e.g. you want to use a different user for the same account). In such a case `lpf logout --delete` will let you completely remove the login information of any account you select from your local machine:
+
+<img class="fancyimage" alt="Functions: cli logout and delete account data" src="img/functions/functions_cli_logout.gif">
+
+
+## Install & Initialize
+**Installation** and **updating** the CLI are both handled using `npm install -g liveperson-functions-cli`. You can also check for updates by using the `lpf version`. Once the CLI has been installed, you can check out the documentation using `lpf help` or head to our [Github Page](https://github.com/LivePersonInc/faas-cli).
+
+{: .notice}
+The CLI is optimized for use with macOS and Linux. If you want to use the CLI from a Windows machine we recommend using the linux subsystem.
```
[the remark this answers is not in the document]
Comment (chrbrt) [reply pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/deep-dive-cli.md:34]: done

**Review by chrbrt — APPROVED** [review]

**Pull request merged**